.github/workflows/sonarcloud.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Uppercase package name
         run: |
           lowercase_package=${{ matrix.package }}
-          lowercase_package=$(echo "$lowercase_package" | tr - _)
+          lowercase_package=$(echo "$lowercase_package" | tr - _ | tr -d plugins/)
```

Comment:
Using `tr` to replace a word might have unintended consequences, I recommend changing this to `sed` instead.